x-pack/plugin/snapshot-repo-test-kit/src/main/java/org/elasticsearch/repositories/blobstore/testkit/RepositoryAnalyzeAction.java

```diff
@@ -488,7 +488,7 @@ public void run() {
                         )
                     )
                 ) {
-                    final int registerOperations = Math.max(nodes.size(), request.getConcurrency());
+                    final int registerOperations = Math.max(nodes.size(), request.getRegisterOperationCount());
```

Comment:
I'm inclined to say no, because I tried describing this in the docs and it just seemed to make it harder to understand. We do more uncontended register ops too because they carry on until the contended ones finish. I'd be amazed if anyone notices this in practice.

Reply:
Eh how about a93f159adb1? Added the word `minimum` to those docs, I think that covers everything simply enough.